Maple2.Server.World/Program.cs

```diff
@@ -48,10 +47,9 @@
 
 builder.Services.AddSingleton<WorldServer>();
 
+
 builder.Host.UseServiceProviderFactory(new AutofacServiceProviderFactory());
 builder.Host.ConfigureContainer<ContainerBuilder>(autofac => {
-    autofac.RegisterType<WorldServer>()
-        .SingleInstance();
     // Database
```

Comment:
no world server in world?

Reply:
It's not needed (at least for now). No other class really should be interacting with WorldServer as currently it's just being used as a class to set events.